Library/Homebrew/formula_installer.rb

```diff
@@ -727,6 +727,7 @@ def build
 
     formula.update_head_version
 
+    puts "Testing formula prefix for emptiness: formula.prefix='#{formula.prefix}'"
```

Comment:
Oops

Reply:
Oops. Fixed.

Sorry for all the hassle here.